data/brands/amenity/restaurant.json

```diff
@@ -2801,6 +2801,24 @@
         "name:zh": "龐德羅莎"
       }
     },
+       {
+      "displayName": "龐德羅莎",
+      "locationSet": {
+        "include": ["tw"]
```

Comment:
in Line 2789, it include 5 country/region: ```"include": ["ae", "eg", "qa", "tw", "us"]```

if you create a new item that suitable for ```"tw"```, you'd better remove it from the list next time.

I'll fix this.

Reply:
Fixed in https://github.com/osmlab/name-suggestion-index/pull/5264/commits/45ad37552569889e78dc761ca9a77405814c59a9